WooCommerce/src/test/kotlin/com/woocommerce/android/ui/sitepicker/sitevisibility/WooSitesVisibilityViewModelTest.kt

```diff
@@ -61,9 +65,8 @@ class WooSitesVisibilityViewModelTest : BaseUnitTest() {
             viewModel.onSiteSelected(WOO_STORE_DEFAULT_UI)
             viewModel.onSiteSelected(WOO_STORE_DEFAULT_UI)
 
-            // Assert
             val updatedState = viewModel.viewState.getOrAwaitValue()
-            assert(updatedState?.wooStores?.first()?.isSelected == true)
-            assert(updatedState?.isSaveButtonEnabled == false)
+            assert(updatedState.wooStores?.first()?.isSelected == true)
```

Comment:
We can simplify this line as you did in the previous function:

```suggestion
            assert(updatedState.wooStores.first().isSelected)
```

Reply:
I fixed this already in a subsequent [PR](https://github.com/woocommerce/woocommerce-android/pull/13188), so I won't update it here to avoid unnecessary commits! But thank for the suggested change :)